source/common/tracing/lightstep_tracer_impl.cc

```diff
@@ -23,6 +23,10 @@ void LightStepSpan::finishSpan(SpanFinalizer& finalizer) {
   span_.Finish();
 }
 
+void LightStepSpan::setOperation(const std::string& operation) {
+  span_.SetOperationName(operation);
```

Comment:
Does this need coverage in lightstep tracer?

Reply:
Not sure best way to test this - the current lightstep tests don't check the operation names passed in via the `startSpan` method, and there are no accessors for the operation name. Any suggestions welcome.

Reply:
If it's not easy to verify it's fine just skip it. I would at least make sure it's called just to have basic crash avoidance testing. (It might already be called not sure).